### PR TITLE
DAC REV B Schematic Review

### PR DESCRIPTION
**Pull request by BrandtBuchda**
This PR is to approve the schematics of the DAC expansion board REV B and merge with the dac-revb-pcb branch. The new DAC has been updated with 8 channel ±10V output voltage range.

## Specifications
Output Voltage:    ±10 V
Channels:    ≥ 8
Update rate:    > 20 kSPS on all channels
Slew Rate:    > 1 V/µs
Output connector type:    BNC connectors
Resolution:    12 bit

## Schematic
The schematic being reviewed is located [here](https://github.com/Severson-Group/AMDC-Hardware/blob/dac-revb-pcb-sch/Accessories/ExpansionBoard_DAC/REV20200627B/DAC_Expansion_Board_Project.pdf)

## PCB Layout
| Estimated Dimensions |   |
|:------------------------:|:-:|
| Board Size (W x H) | 3.9" x 3.9"|
| Number of Layers | 2 |

image

## Bill of Materials
In quantity of 10, the components cost $60 per board on Digi-Key. The BOM can be found [here](https://github.com/Severson-Group/AMDC-Hardware/blob/dac-revb-pcb-sch/Accessories/ExpansionBoard_DAC/REV20200627B/DAC_bom_qty_10.md)

### CODE REVIEW

**Comment by BrandtBuchda** [comment]
### Change list from Schematic Review

- [x] 1. Add columns with pinout descriptions to DB-15 connector schematic. Use net labels for power signals coming from the connector.
- [x] 2. Add jumpers for power signals that overlap between isoSPI and the power stack connectors
- [x] 3. Add DNP to R1 and resistance value to R2
- [x] 4. Add a note indicating that R_B1 and R_B2 are labeled in this manner to correspond with the isoSPI datasheet
- [x] 5. Tie MSTR pin on U1 to GND
- [x] 6. Update all 5V pins to be +5V
- [x] 7. Vertically align power pins
- [x] 8. Remove R5 from isoSPI transformer. Look for single transformer package to replace it.
- [x] 9. Add output capacitor to voltage regulators U13 and U16
- [x] 10. Look into fixed voltage LDO to replace U13, U14, U16
- [x] 11. Tie unused isolator pins on U17 to GND or find 3 channel replacement.
- [x] 12. Add NC to all unused pins
- [x] 13. Replace RC filters with discrete RC components.
- [x] 14. Add label / symbol to ESD protection
- [x] 15. Reduce the number of unique resistors used.
- [x] 16. Change AMDC_GND to GND on U1
- [x] 17. Add missing GND to FL5
- [x] 18. Fix short between pins 7, 8, and 15 of CON1
- [x] 19. Add decoupling capacitors to opamp power inputs
- [x] 20. Fix U12 to only isolate the status lines from the power stack
- [x] 21. Update OP Amp resistors to have 0.1% tolerances

**Review by npetersen2 — CHANGES_REQUESTED** [review]
@BrandtBuchda @elsevers @mimayberry 

Looks pretty good to me, I have a few more comments that we may want to address, otherwise we can decide to ignore them and move on (i.e. maybe these are optimizations for REV C):

1. The BOM of $90 seems like a lot to me for a simple DAC board... So I dug into what was so expensive.

> - **Op amps:** the op amps run $20 per board, which seems steep for generic op amps. Two ideas: 1) combine them into larger packages (i.e. 4x per IC) to reduce cost. 2) Keep 1x per IC, but use a cheaper op amp that meets specs. The op amp in the schematics costs ~$2.50 each, while [this](https://www.digikey.com/product-detail/en/on-semiconductor/NCS20071SN2T1G/NCS20071SN2T1GOSCT-ND/7556484) other op amp meets specs and costs ~$0.50 each. I believe your op amp is so expensive since it has a high slew rate (much higher than spec).
> - **DC/DCs:** the power conversion / filtering stage costs ~$15 per board. This isn't bad, but after a quick Digi-Key search, I found an "identical" spec +/- 15V DC/DC that costs 50% as much as the schematic part, [see here](https://www.digikey.com/product-detail/en/cui-inc/PDME1-S5-D15-S/102-6323-ND/10229858). Similar story for the 5V => 9V DC/DC, [see here](https://www.digikey.com/product-detail/en/cui-inc/PDSE1-S5-S9-S/102-6292-ND/10229827).
> - **BNC connectors:** I see you picked 75 ohm BNC connectors. Why 75 ohms? @mimayberry do we have a preference for impedance / does it matter? I think most BNC cables are 50 ohms. ...upon some research, looks like below 10MHz it doesn't matter... so this probably doesn't matter.

2. Mechanical parts on schematic / BOM: please add stand-offs + screws to your schematic / BOM (see `AMDC-Hardware` for an example). This way, when a user orders the BOM, they automatically get the required mechanical parts.

3. Labeling clean-up on schematics: can you move around a few labels to make it a little cleaner:

> - BNC J1..8: the part number is on top of the GND connection, please move it to the top of the connector box, and move the the designator 100mil higher.
> - R4 goes into a wire

4. Rename the isoSPI output SPI lines to not prepend `LV_` -- on our schematics, we typically use `LV_` to indicate a low voltage version of the signal. Maybe use `isoSPI_` to show it's coming from isoSPI.

I'll let others decide which of these to implement and which to save for REV C. Otherwise, schematic looks good; ready to approve and move onto PCB design + ordering.

**Review by mimayberry — COMMENTED** [review]
This is all looking pretty good to me. A few comments about @npetersen2 's   suggestions 
1. For the op-amps I would suggest going to quad packages so only two are required. I always feel less components the better. Also, I think it's worth spending the extra money on the high-performance op-amp. The noise performance and accuracy are a good deal better than the cheaper one.
2. I'm okay going with the less expensive DC-DC convertors (both 15V and 9V), the output ripple is ~20% higher for the cheaper ones, though this shouldn't matter much as they're going through LDOs after. However, the cheaper 9V option has a limited number in-stock, I would keep a watch on that.
3. I agree with Nathan's suggestions for cleaning up the schematics and renaming the output SPI lines.

File: altium/AMDC.PrjPCBStructure
Comment (npetersen2) [inline altium/AMDC.PrjPCBStructure]: @BrandtBuchda Is there a reason that the `AMDC.PrjPCBStructure` file has changed for the DAC expansion board? I think this is a mistake.

**Review by npetersen2 — CHANGES_REQUESTED** [review]

**Review by npetersen2 — CHANGES_REQUESTED** [review]
@BrandtBuchda Looks great!!! Only one last comment then I am ready to approve:

- Please make the mounting holes "Standard (No BOM)" so they don't appear in the BOM file and then update the BOM files accordingly.

Once this is done, please rerequest a review from me -- I will then approve.

**Comment by BrandtBuchda** [comment]
> @BrandtBuchda Looks great!!! Only one last comment then I am ready to approve:
> 
> * Please make the mounting holes "Standard (No BOM)" so they don't appear in the BOM file and then update the BOM files accordingly.
> 
> Once this is done, please rerequest a review from me -- I will then approve.

Should I do the same for the DNP resistor?

**Comment by npetersen2** [comment]
@BrandtBuchda yeah, let's remove the DNP resistor from the BOM as well.

**Review by mimayberry — APPROVED** [review]
I'm happy with this, I think everything looks great

**Review by elsevers — APPROVED** [review]
Looks good to me. I think you can move onto the PCB design phase.

A couple items I'd like to see you address in the PCB PR:
1. The BOM strikes me as odd. The file name is `DAC_bom_qty_10.md`, yet the quantity column indicates quantity 1.

2. Your 3D board rendering does not seem to be updated to include the new jumpers (`JP6`-`JP8`)

3. If possible, I suggest that you implement `JP6` - `JP8` in a way so that these block jumpers that you are using for `JP1` - `JP5` also operate on `JP6`- `JP8`. Can you get large enough jumper blocks for this?

**Review by npetersen2 — APPROVED** [review]
Looks good!

**Comment by npetersen2** [comment]
Feel free to click the merge button and then delete the branch.

**Pull request merged**
